src/app/ad-authoring/ad-authoring.state.ts

```diff
@@ -9,15 +9,177 @@ export interface AdAuthoringWorkflowState {
+      base64AssetStr +
+      '\\"><\\/amp-img><\\/body><\\/html>"';
+
+    // if the file uploaded is a video (not image)
+    if (assetFile != null && assetFile.type.includes('video')) {
+      adAMPHTML =
+        '"<!doctype html><html amp4ads><head><meta charset=\\"utf-8\\"><meta name=\\"viewport\\" content=\\"width=device-width,minimum-scale=1\\"><meta name=\\"amp-cta-type\\" content=\\"' +
+        callToActionStr +
+        '\\"><meta name=\\"amp-cta-url\\" content=\\"' +
+        landingPageUrl +
+        '\\"><meta name=\\"amp-cta-landing-page-type\\" content=\\"' +
+        landingPageType +
+        '\\"><style amp4ads-boilerplate>body{visibility:hidden}<\\/style><script async src=\\"https:\\/\\/cdn.ampproject.org\\/amp4ads-v0.js\\"><\\/script><script async custom-element=\\"amp-video\\" src=\\"https:\\/\\/cdn.ampproject.org\\/v0\\/amp-video-0.1.js\\"><\\/script><\\/head><body><p>Hello, fake ad with srcdoc<\\/p><amp-video layout=\\"fill\\" height=\\"1920\\" width=\\"1080\\" autoplay loop> <source src=\\"' +
+        base64AssetStr +
+        '\\" type=\\"' +
+        assetFile.type +
+        '\\" \\/><\\/amp-video><\\/body><\\/html>"';
+    }
+
+    const storyAMPHTML =
+      `<!DOCTYPE html>
+    <html amp="🤠-invalid" lang="en">
+      <head>
+        <meta charset="utf-8" />
+        <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+        <script async src="https://cdn.ampproject.org/v0.js"></script>
+        <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+        <script async custom-element="amp-story-auto-ads" src="https://cdn.ampproject.org/v0/amp-story-auto-ads-0.1.js"></script>
+        <title>AMP Story</title>
+        <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1" />
+        <link rel="canonical" href="/" />
+      </head>
+      <body>
+        <amp-story standalone>  
+          <amp-story-auto-ads id="i-amphtml-demo-1" development>
+            <script type="application/json">
+              {
+                "ad-attributes": {
+                  "type": "fake",
```

Comment:
We shouldn't need to concatenate here since we're using a template string. Instead use ${adAmpHtml}